internal/utils_test.go

```diff
@@ -0,0 +1,43 @@
+package internal
+
+import (
+	"testing"
+)
+
+func TestGetSemVer(t *testing.T) {
+	if sdkVersion, e := GetSemVer(""); sdkVersion != 0 || e != nil {
+		t.Error("Expected sdk version to be 0 but got : ", sdkVersion)
+	}
```

Comment:
Use `assert.Equal` and `assert.Nil` the code will be more clear.

Reply:
Thanks for the suggestion, done.